test_sets/External_UCS-related_params.txt

```diff
@@ -0,0 +1,22 @@
+./test_scripts/Polices/External_UCS/001_ATF_P_TC_PreloadedPT_without_external_consent_status_groups_struct.lua Check_that_SDL_starts_successfully_in_case_external_consent_status_groups_struct_is_omitted_in_PreloadedPT
```

Comment:
@dboltovskyi, thank you for clarification.
As I search in JIRA, the question is "APPLINK-28543 is missing in Specification". Please link it to Testing coverage subtask to have traceability.

According to link, today is also working, seems some i-net problems.